discovery/db_discovery.go

```diff
@@ -193,10 +212,17 @@ func ethOrchToDBOrch(orch *lpTypes.Transcoder) *common.DBOrch {
 	if orch == nil {
 		return nil
 	}
-	return &common.DBOrch{
+	dbO := &common.DBOrch{
 		ServiceURI:   orch.ServiceURI,
 		EthereumAddr: orch.Address.String(),
 	}
+	if orch.ActivationRound != nil {
```

Comment:
I don't think `ActivationRound` and `DeactivationRound` will ever be nil since the default value returned from the contract will be 0.

Reply:
fixed in 74f273e